libraries/botbuilder-ai/botbuilder/ai/qna/models/__init__.py

```diff
@@ -13,6 +13,9 @@
 from .query_result import QueryResult
 from .query_results import QueryResults
 from .train_request_body import TrainRequestBody
+from .prompt import Prompt
```

Comment:
Simple house cleaning; could you please keep the imports in alphabetical order, as well as the strings in `__all__` array?

Reply:
Done